vignettes/r2dii-analysis.Rmd

```diff
@@ -46,7 +40,7 @@ matched <- match_name(
 ) %>%
   prioritize()
 
-matched %>% head()
+matched
```

Comment:
If tibble or dplyr is attached, then tibbles will print only 10 rows. Thus `head()` seems needless.